src/v3/test/integration/authenticator-enroll-security-question.test.tsx

```diff
@@ -103,6 +106,266 @@ describe('authenticator-enroll-security-question', () => {
+              answer,
+            },
+            stateHandle: 'fake-stateHandle',
+          }),
+          headers: {
+            Accept: 'application/json; okta-version=1.0.0',
+            'Content-Type': 'application/json',
+            'X-Okta-User-Agent-Extended': 'okta-auth-js/9.9.9',
+          },
+          withCredentials: true,
+        },
+      );
+      const answerEleError = await findByTestId('credentials.answer-error');
+      // Previously entered characters should remain in the field
+      expect((await findByTestId('credentials.answer') as HTMLInputElement).value).toBe(answer);
+      expect(answerEleError.innerHTML).toEqual('The security question answer must be at least 4 characters in length');
+    });
+
+    it('should show field level character count error message on multiple attempts to submit with invalid character count', async () => {
+      const {
+        user, authClient, findByText, findByTestId,
+      } = await setup({ mockResponse: predefinedQuestionAnswerErrorResponse });
+
+      await findByText(/Set up security question/);
+      const submitButton = await findByText('Verify', { selector: 'button' });
+      const answerEle = await findByTestId('credentials.answer') as HTMLInputElement;
+
+      const answer = 'pi';
+      await user.type(answerEle, answer);
+      expect(answerEle.value).toEqual(answer);
+
+      await user.click(submitButton);
+      expect(authClient.options.httpRequestClient).toHaveBeenCalledWith(
+        'POST',
+        'https://oie-4695462.oktapreview.com/idp/idx/challenge/answer',
+        {
+          data: JSON.stringify({
+            credentials: {
+              questionKey: 'disliked_food',
+              answer,
+            },
+            stateHandle: 'fake-stateHandle',
+          }),
+          headers: {
+            Accept: 'application/json; okta-version=1.0.0',
+            'Content-Type': 'application/json',
+            'X-Okta-User-Agent-Extended': 'okta-auth-js/9.9.9',
+          },
+          withCredentials: true,
+        },
+      );
+      const answerEleError = await findByTestId('credentials.answer-error');
+      expect(answerEleError.innerHTML).toEqual('The security question answer must be at least 4 characters in length');
+      await user.click(await findByText('Verify', { selector: 'button' }));
+      expect(authClient.options.httpRequestClient).toHaveBeenCalledWith(
+        'POST',
+        'https://oie-4695462.oktapreview.com/idp/idx/challenge/answer',
+        {
+          data: JSON.stringify({
+            credentials: {
+              questionKey: 'disliked_food',
+              answer,
+            },
+            stateHandle: 'fake-stateHandle',
+          }),
+          headers: {
+            Accept: 'application/json; okta-version=1.0.0',
+            'Content-Type': 'application/json',
+            'X-Okta-User-Agent-Extended': 'okta-auth-js/9.9.9',
+          },
+          withCredentials: true,
+        },
+      );
```

Comment:
Let's add a tech-debt/todo to extract/DRY this boilerplate to a utility function.